lib/mollie/settlement.rb

```diff
@@ -2,12 +2,14 @@ module Mollie
   class Settlement < Base
     attr_accessor :id,
                   :reference,
-                  :settled_datetime,
+                  :created_at,
+                  :settled_at,
+                  :status,
```

Comment:
A settlement has different statuses:
- open
- pending
- paidout
- failed

Can you add methods like `open?`, `pending?`, `paidout?` and `failed?`